src/test/scala/is/hail/methods/LinearRegressionBurdenSuite.scala

```diff
@@ -1,14 +1,31 @@
 package is.hail.methods
 
 import is.hail.SparkSuite
+import is.hail.TestUtils.interceptFatal
 import is.hail.expr.TDouble
 import is.hail.keytable.KeyTable
 import is.hail.utils._
 import is.hail.variant.VariantDataset
 import org.apache.spark.sql.Row
 import org.testng.annotations.Test
 
+object BurdenSuiteUtils {
```

Comment:
Put this in a separate file, it's not specific to linear or logistic so it doesn't belong in either of the existing files..